### DISCUSSION TITLE
[Feature Request] Support for Pull Request Generators

### DISCUSSION THREAD

**Discussion by gtaylor5** [katherinepeterson/readme.so#217] [Ideas]
## Problem (Not Really)
`readme.so` is such an amazing tool. However, I find that I don't need to write project-level `ReadMes` very often. Instead, I find myself needing to write PRs or Issues (like this one) almost every day. It might be out of scope of this project but would be an awesome feature / add-on

**Comment by octokatherine** [comment]
I love this idea, but it feels a little out of scope of this project.

I suggested it to a friend of mine though and he's working on building it out as a separate project so stay tuned! cc @charlesdesiderio
